### ISSUE TITLE
ofVideoPlayer and Emscripten

### ISSUE THREAD

**Issue by Jonathhhan** [openframeworks/openFrameworks#7377]
With the current OF branch the videoPlayer examples only work with Emscripten, if I use `ofSoundStream()` in `ofSetup()`. Somehow that was not needed before...

Thats the error without `ofSoundStream`:
```
Uncaught ReferenceError: AUDIO is not defined
    at _html5video_player_create (videoPlayerExample.js:1:183919)
    at ofxEmscriptenVideoPlayer::ofxEmscriptenVideoPlayer() (videoPlayerExample.wasm:0x4605f)
    at ofApp::setup() (videoPlayerExample.wasm:0x908f4)
```

**Comment by dimitre** [comment]
Is this some regression? it used to work well? 
if yes can you please try to identify which commit introduced this behavior?

**Comment by Jonathhhan** [comment]
@dimitre yes, it was working before (not long ago). will try to find the commit the next days...

**Comment by Jonathhhan** [comment]
@dimitre still the same with the latest master. I will have a look into the recent commits, when I have some time (soon).

**Comment by Jonathhhan** [comment]
@dimitre I found it: https://github.com/openframeworks/openFrameworks/pull/7301/files :)

**Comment by Jonathhhan** [comment]
@2bbb is there a way to enable soundstream for the videoplayer automatically (like before) with Emscripten? Thank you.

**Comment by 2bbb** [comment]
Hi, @Jonathhhan 

maybe there is a way, I think.
but I don't understand about Emscripten...

**Comment by 2bbb** [comment]
Notice: ***I DON'T UNDERSTAND Emscripten.***

can we call `ofxEmscriptenAudioContext()` in `ofxEmscriptenVideoPlayer:: load()`? (isn't there `setup`...?)
currently `ofxEmscriptenAudioContext()` is hide in implementation only in `ofxEmscriptenSoundPlayer.cpp`.
but maybe we can test quickly with dirty way to edit `ofxEmscriptenVideoPlayer.cpp` like:

```cpp
extern int ofxEmscriptenAudioContext();

bool ofxEmscriptenVideoPlayer::load(string name){
	ofxEmscriptenAudioContext();
	if (name.substr(0, 12) == "blob:http://" || name.substr(0, 13) == "blob:https://"){
  ....
}
```

this way is to only for test. if it clear problem, then we need to decide to make ofxEmscriptenAudioContext public in header, or not.

**Comment by Jonathhhan** [comment]
@2bbb thanks a lot. I tried your suggestion but it does not work.
No there is no setup (like in `ofVideoPlayer`?)
Now the error is:
`Uncaught TypeError: Cannot read properties of undefined (reading 'createMediaElementSource')`

**Comment by ofTheo** [comment]
Sort of a hack but this works for me: 

https://gist.github.com/ofTheo/4283463783876f901f80c00f99bc56ec
